babel.config.js

```diff
@@ -10,7 +10,12 @@ const presets = [
             ]
         },
         useBuiltIns: "usage",
-    }]
+    }],
+    "@babel/preset-react"
 ];
 
-module.exports = { presets };
\ No newline at end of file
+const plugins = [
+    "@babel/plugin-syntax-dynamic-import"  
+];
+
+module.exports = { presets };
```

Comment:
Moving config away from .babelrc

Reply:
The new version of `react-docgen` uses the babel configuration but only knows about this file